dev-previews/dynamic-metrics-collection/README.md

```diff
@@ -0,0 +1,68 @@
+## Dynamic Metric Collection (Custom Metrics Collection)
+
+Dynamic metrics collection refers to the ability to trigger collecting metrics on managed clusters based on specific conditions. Collecting metrics consumes resources on hub cluster. This is especially important when considering collecting metrics across a large fleet of clusters. It makes sense to start collect certain metrics only when they are likely going to be needed for optmially using resources. When problems occur on a managed cluster, it may be necessary to collect metrics at a higher rate to help analyze the problems. Dynamic metrics collection enables both these use cases. Metrics collection stops automatically 15 minutes after the underlying condition no longer exists.
+
+Dynamic metric collection is supported with a group of collection rules (`collection_rules`). A collection rule is a named entity that specifies
```

Comment:
A collection rule is a named entity that specifies the following attributes:

Reply:
Updated